components/brave_extension/extension/brave_extension/content_cosmetic.ts

```diff
@@ -553,7 +553,7 @@ const scheduleQueuePump = (hide1pContent: boolean, generichide: boolean) => {
   }, { timeout: maxTimeMSBeforeStart })
 }
 
-const vettedSearchEngines = ['duckduckgo', 'qwant', 'bing', 'startpage', 'yahoo', 'onesearch', 'google', 'yandex']
+const vettedSearchEngines = ['duckduckgo', 'qwant', 'bing', 'startpage', 'yahoo', 'onesearch', 'google', 'yandex', 'ecosia']
```

Comment:
I think yahoo should be removed from here, right?

@SergeyZhukovsky , just a heads up to make sure the Android work you're doing stays in sync with the above changes

Reply:
sounds good, I added ecosia as well